src/Codeception/Module.php

```diff
@@ -288,6 +288,18 @@ protected function debugSection($title, $message)
         $this->debug("[$title] $message");
     }
 
+    /**
+     * Short text message to an amount of chars
+     *
+     * @param $message
+     * @param $chars
+     * @return string
+     */
+    protected function shortenMessage($message, $chars = 150)
+    {
+        return substr($message, 0, $chars) . "...";
```

Comment:
1. You should only add  `...` if string was longer than 150.
2. You should use mb_substr with `utf-8` charset instead of substr, because substr breaks utf-8 characters and Codeception doesn't work with anything else than utf-8. All `substr` and `strlen` calls were replaced with mb equivalents in the course of multiple pull requests to fix various issues.

A good similar example is at https://github.com/Codeception/Codeception/blob/2.5.5/src/Codeception/Step.php#L126-L127